### PR TITLE
single network services/endpoints can not have duplicate address

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/ops/deployment/deployment-models/index.md [hunk 1]
```diff
@@ -197,7 +197,8 @@ fits your existing network topology.
 In the simplest case, a service mesh operates over a single fully connected
 network. In a single network model, all
 {{< gloss "workload instance" >}}workload instances{{< /gloss >}}
-can reach each other directly without an Istio gateway.
+can reach each other directly without an Istio gateway. In addition, for single network across multi-clusters,
+Istio requires services and endpoints cannot have overlapping IP addresses.
 
 A single network allows Istio to configure service consumers in a uniform
 way across the mesh with the ability to directly address workload instances.
```
Comment (frankbu) [inline content/en/docs/ops/deployment/deployment-models/index.md:204]:
```suggestion
can reach each other directly without an Istio gateway.

A single network allows Istio to configure service consumers in a uniform
way across the mesh with the ability to directly address workload instances.
Note, however, that for a single network across multiple clusters,
services and endpoints cannot have overlapping IP addresses.
```

File: content/en/docs/ops/deployment/deployment-models/index.md [hunk 2]
```diff
@@ -201,6 +201,8 @@ can reach each other directly without an Istio gateway.
...
+Note, however, that for a single network across multiple clusters,
```
Comment (howardjohn) [inline content/en/docs/ops/deployment/deployment-models/index.md:204]: Definitely endpoints is true, I am not 100% sure we cannot have overlapping Service. I think @ramaraochavali may have mentioned this?

Comment (hzxuzhonghu) [reply content/en/docs/ops/deployment/deployment-models/index.md:204]: Otherwise a call to service A can be routed to serviceB, if via VIP.

Comment (ramaraochavali) [reply content/en/docs/ops/deployment/deployment-models/index.md:204]: Yes. ClusterIP has to be unique in a single network.

**Review by frankbu — APPROVED** [review]

**Pull request merged**